### PR TITLE
added port option to server.py script

### PR DESCRIPTION
**Pull request by kikocorreoso**
I added a port option to the command line in order to run `server.py` on other ports different than 8000 without the need to hard code `server.py`.

### CODE REVIEW

File: www/doc/en/dev_env.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ A web server is necessary to test the scripts locally while developing. Any web
 server that can serve files with the brython_directory/www as document root will 
 work ; you can use the built-in web server provided in the distribution : open 
 a console window, go to the directory, and run `python server.py`. This will 
-start the server on port 8000 (edit _server.py_ to change the port number).
+start the server on port 8000 (if you want to use other port number you can use,
```
Comment (rovitotv) [inline www/doc/en/dev_env.md]: I recommend stating it this way:
start the server on port 8000 (if you want to use a different port number you can use,

File: www/doc/en/dev_env.md [hunk 2]
```diff
@@ -10,7 +10,8 @@ A web server is necessary to test the scripts locally while developing. Any web
 server that can serve files with the brython_directory/www as document root will 
 work ; you can use the built-in web server provided in the distribution : open 
 a console window, go to the directory, and run `python server.py`. This will 
-start the server on port 8000 (edit _server.py_ to change the port number).
+start the server on port 8000 (if you want to use other port number you can use, 
+for instance, `python server.py --port 8001` to use port 8001).
```
Comment (rovitotv) [inline www/doc/en/dev_env.md]: `python server.py --port 8001` to use port 8001).

**Comment by rovitotv** [comment]
I think this is an excellent idea and will make it easier for new users to get started.  I like that you made the argument optional.  The code changes look good to me, I did add a few comments on the documentation updates.

**Comment by kikocorreoso** [comment]
Updated, aaaahhhh, english is more difficult than js or python.

**Comment by rovitotv** [comment]
That update looks perfect to me!  I think you have improved the documentation tremendously for a new user.

**Comment by PierreQuentel** [comment]
Thanks to both of you. I have updated the documentation in French too.

**Pull request merged**